x-pack/legacy/plugins/apm/e2e/ci/prepare-kibana.sh

```diff
@@ -1,24 +1,21 @@
 #!/usr/bin/env bash
 set -e
 
-CYPRESS_DIR="x-pack/legacy/plugins/apm/e2e"
+E2E_DIR="x-pack/legacy/plugins/apm/e2e"
 
 echo "1/3 Install dependencies ..."
 # shellcheck disable=SC1091
 source src/dev/ci_setup/setup_env.sh true
 yarn kbn bootstrap
-cp ${CYPRESS_DIR}/ci/kibana.dev.yml config/kibana.dev.yml
-echo 'elasticsearch:' >> config/kibana.dev.yml
-cp ${CYPRESS_DIR}/ci/kibana.dev.yml config/kibana.yml
 
 echo "2/3 Ingest test data ..."
-pushd ${CYPRESS_DIR}
+pushd ${E2E_DIR}
 yarn install
 curl --silent https://storage.googleapis.com/apm-ui-e2e-static-data/events.json --output ingest-data/events.json
-node ingest-data/replay.js --server-url http://localhost:8200 --secret-token abcd --events ./events.json > ingest-data.log
+node ingest-data/replay.js --server-url http://localhost:8201 --secret-token abcd --events ./events.json > ingest-data.log
 
 echo "3/3 Start Kibana ..."
 popd
 ## Might help to avoid FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 export NODE_OPTIONS="--max-old-space-size=4096"
-nohup node scripts/kibana --no-base-path --csp.strict=false --optimize.watch=false> kibana.log 2>&1 &
+nohup node scripts/kibana --config "${E2E_DIR}/ci/kibana.e2e.yml" --no-base-path --optimize.watch=false> kibana.log 2>&1 &
```

Comment:
What's the difference with https://github.com/elastic/kibana/pull/59152/files#diff-ccfe59ee0ec66fe86f6596aaf1fce60cR24? Are those flags also required in the CI?

Reply:
**This starts in dev mode**
```
node ./scripts/kibana --no-base-path --dev --no-dev-config --config x-pack/legacy/plugins/apm/e2e/ci/kibana.e2e.yml
```

**This starts in prod mode**
```
node scripts/kibana --config "${E2E_DIR}/ci/kibana.e2e.yml" --no-base-path --optimize.watch=false
```

I think we should choose whatever is faster. I guess we can test this on CI by running it 100 times or so.
 I imagine dev compiles faster (because it doesn't need to compress resources) but prod is faster to load because assets are smaller. So could go both ways.

The reason I opted for using dev mode locally is because this is what developers is normally running, and I saw some issues when running prod mode without cleaning up after having run dev mode.

Reply:
Thanks for the clarification, I'd say to use the same as the local one to avoid any misleading, then it will be easy to reproduce it anyway. Thanks again